examples/mutators/search/README.md

```diff
@@ -0,0 +1,167 @@
+# search: Examples
```

Comment:
Where does this top-level README show up? We only have two types of docs:

- Per example package docs
- Commmand reference

I think this is the latter? If so, I think it's fine to have this in a separate `md` file which is returned by `usage` func in code and hence --help flag.

@mengqiy If you agree we need to have a utility function in Go that returns the content of the README file.

Reply:
Yes. I feel having the usage as a long string in main.go looks weird. Having it as a usage.md from where go can return it is a good way of doing it. That is how if it done in kpt currently.

Reply:
That sounds like a good idea.
I imagine we need to strip the markdown syntax (e.g. "```" and "#") before feeding it to `--help`. Otherwise, it looks weird.

Reply:
We can leverage `go generate` similar to kpt https://github.com/GoogleContainerTools/kpt/blob/master/Makefile#L40

Reply:
Removed this doc for now. We need to implement the `make generate` similar to kpt so that we can have detailed command reference git docs and parts of it can be rendered as generated godocs.

Reply:
If we have `usage.md`, then kpt fn doc --help will become less useful.
If we create utility functions for go, we should do it for ts functions too. So every fn has a `usage.md`.